src/Select/SelectDirective.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Nuwave\Lighthouse\Select;
+
+use Nuwave\Lighthouse\Schema\Directives\BaseDirective;
+
+class SelectDirective extends BaseDirective
+{
+    public static function definition(): string
+    {
+        return /** @lang GraphQL */ <<<'GRAPHQL'
+"""
+Specify the SQL column dependencies of this field.
+"""
+directive @select(
+  """
+  SQL columns names to pass to the Eloquent query builder
+  """
+  columns: [String!]
```

Comment:
```suggestion
  columns: [String!]!
```

This directive has no use without this argument.